go.mod

```diff
@@ -25,6 +26,7 @@ require (
 	github.com/Masterminds/squirrel v1.5.4
 	github.com/PaesslerAG/gval v1.2.2
 	github.com/PaesslerAG/jsonpath v0.1.1
+	github.com/anicoll/screamer v0.0.0-20241206035431-9ba919b54dfd
```

Comment:
We try not to take dependencies on libraries that aren't stable.

Reply:
Based on this comment and the one below.

raised this PR.
https://github.com/redpanda-data/connect/pull/3083